### PR TITLE
Visitation batch processor daemons

### PR DESCRIPTION
**Pull request by xbrianh**
Connects to #614 #539
Depends on #735

### Test plan
Unit and integration tests.

### CODE REVIEW

File: daemons/dss-visitation-sentinel/.chalice/config.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      "api_gateway_stage": "stochastic",
+      "environment_variables": {
+        "DSS_DEPLOYMENT_STAGE": "stochastic",
+        "DSS_ES_ENDPOINT": "search-dss-index-stochastic-yoahuyrbtn2jqbzgaeqbnx673i.us-west-2.es.amazonaws.com",
+        "DSS_S3_BUCKET": "dss-stochastic",
+        "DSS_GS_BUCKET": "dss-stochastic",
+        "API_HOST": "dss.dev.data.humancellatlas.org",
+        "DSS_SUBSCRIPTION_AUTHORIZED_DOMAINS": "chanzuckerberg.com ucsc.edu broadinstitute.org ebi.ac.uk human-cell-atlas-travis-test.iam.gserviceaccount.com broad-dsde-mint-dev.iam.gserviceaccount.com broad-dsde-mint-staging.iam.gserviceaccount.com",
+        "DSS_NOTIFICATION_SENDER": "DSS Notifier <noreply@humancellatlas.org>",
+        "DSS_S3_CHECKOUT_BUCKET": "org-humancellatlas-dss-checkout-dev"
```
Comment (kislyuk) [inline daemons/dss-visitation-sentinel/.chalice/config.json]: All of these variables seem like they are set by the deployer, please remove them.

Comment (xbrianh) [reply daemons/dss-visitation-sentinel/.chalice/config.json]: fixed.

**Comment by xbrianh** [comment]
@kislyuk IAM policy has been restricted.

**Comment by hannes-ucsc** [comment]
Commit message is a bit monosyllabic.

**Comment by hannes-ucsc** [comment]
Superseded by #792.

**Pull request closed without merging**